src/main/resources/application.yml

```diff
@@ -37,6 +37,13 @@ spring.thymeleaf.prefix: file:${osiam.home}/templates/web/
 spring.jpa.hibernate.ddl-auto: none
 server.port: ${osiam.port:8080}
 
+management.context-path: /monitor
+info.app:
+  encoding: @project.build.sourceEncoding@
+  java:
+    source: @java.version@
+    target: @java.version@
```

Comment:
I don't think we need to expose this kind of information. [This](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready-application-info-build) looks far more interesting. But it's better to add it, after the Gradle migration is done.